package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -81,11 +81,13 @@
 HydrogenBondLifetimes
 ~~~~~~~~~~~~~~~~~~~~~
 
-The class Analyzing hydrogen bond lifetimes (HBL)
-:class:`HydrogenBondLifetimes` has been removed.
-Please use the newer module
-:mod:`MDAnalysis.analysis.hydrogenbonds.hbond_analysis`.
+To analyse hydrogen bond lifetime, use
+:mod:`MDAnalysis.analysis.hydrogenbonds.hbond_analysis` and
+its method `.lifetime` to find the hydrogen bonds and their lifetime.
 
+See Also
+    --------
+    :mod:`MDAnalysis.analysis.hydrogenbonds.hbond_analysis`
```

Comment:
This tab/space might be responsible for the failing sphinx build tests.
```suggestion
--------
:mod:`MDAnalysis.analysis.hydrogenbonds.hbond_analysis`


.. versionchanged:: 2.0.0
   The ``HydrogenBondLifetimes`` analysis class was removed.
```